samples/client/petstore/java/jersey1/src/test/java/io/swagger/client/ApiClientTest.java

```diff
@@ -40,15 +40,24 @@ public void testParseAndFormatDate() {
 
     @Test
     public void testIsJsonMime() {
-      assertFalse(apiClient.isJsonMime(null));
-      assertFalse(apiClient.isJsonMime(""));
-      assertFalse(apiClient.isJsonMime("text/plain"));
-      assertFalse(apiClient.isJsonMime("application/xml"));
-      assertFalse(apiClient.isJsonMime("application/jsonp"));
-
-      assertTrue(apiClient.isJsonMime("application/json"));
-      assertTrue(apiClient.isJsonMime("application/json; charset=UTF8"));
-      assertTrue(apiClient.isJsonMime("APPLICATION/JSON"));
+        assertFalse(apiClient.isJsonMime(null));
+        assertFalse(apiClient.isJsonMime(""));
+        assertFalse(apiClient.isJsonMime("text/plain"));
+        assertFalse(apiClient.isJsonMime("application/xml"));
+        assertFalse(apiClient.isJsonMime("application/jsonp"));
+        assertFalse(apiClient.isJsonMime("example/json"));
+        assertFalse(apiClient.isJsonMime("example/foo+bar+jsonx"));
+        assertFalse(apiClient.isJsonMime("example/foo+bar+xjson"));
+
+        assertTrue(apiClient.isJsonMime("application/json"));
+        assertTrue(apiClient.isJsonMime("application/json; charset=UTF8"));
+        assertTrue(apiClient.isJsonMime("APPLICATION/JSON"));
+
+        assertTrue(apiClient.isJsonMime("application/json\t"));
+        assertTrue(apiClient.isJsonMime("example/foo+bar+json"));
+        assertTrue(apiClient.isJsonMime("example/foo+json;x;y"));
+        assertTrue(apiClient.isJsonMime("example/foo+json\t;"));
+        assertTrue(apiClient.isJsonMime("Example/fOO+JSON"));
```

Comment:
If you want a real-live example, `application/problem+json` (from [RFC 7807](https://tools.ietf.org/html/rfc7807)) is often used in APIs.